packages/screens/Mini/Profile/ProfileScreen.tsx

```diff
@@ -88,6 +88,11 @@ export const ProfileScreen: ScreenFC<"MiniProfile"> = ({ navigation }) => {
     });
   };
 
+  // disable multiple wallet creation, remove to re enable multiple
```

Comment:
This comment is clear enough to me if I read this code here  but in some case if someone debug and follow the code, he can see that var is const then believe that its value will be immutable but not in this case.

so maybe we should create 2 consts :

const profileScreen = [<settings>, <dapps>] 
const profileForMultiWalletsScreen = [<add account>, ... profileScreen];